### DISCUSSION TITLE
Can't get verified because the domain for the school is incorrect

### DISCUSSION THREAD

**Discussion by pgasperowicz** [community/community#102929] [GitHub Education]
### Hi everyone,

When applying, I receive the following errors:

> We require applicants of Medical University of Warsaw to use one of these school-issued email addresses to apply:
> - student.wum.edu.pl
> - wunn.edu.pl

Second domain is "wunn.edu.pl" with double-n. Should be "wum.edu.pl"
Is there a way to get this updated?

Exact same problem: [https://github.com/orgs/community/discussions/41457](url)

**Comment by samus-aran** [comment]
Hi @pgasperowicz 

A little late to responding but I wanted to direct you to how you can resolve this. I'll keep this discussion close as the guidance should direct you to reach out to our support team.

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
